tests/test_build_html.py

```diff
@@ -24,6 +24,12 @@
 from sphinx.util import docutils, md5
 from sphinx.util.inventory import InventoryFile
 
+if docutils.__version_info__ < (0, 17):
+    FIGURE_CAPTION = ".//div[@class='figure align-default']/p[@class='caption']"
+else:
+    FIGURE_CAPTION = ".//figure/figcaption/p"
```

Comment:
Since v0.17, docutils starts to use `<figure>` and `<figcaption>` for figure and its caption on HTML output.